### PR TITLE
docs: reflect new state of CD in readme

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

updates readme to match latest CD process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,21 +47,10 @@ If your target environment does not support that, make sure you transpile this p
...
 
-### Commit messages and releasing to NPM
+### Releases
 
-_tl;dr_
+Commits that are merged to `main` on GitHub will be analysed by [semantic-release](https://semantic-release.gitbook.io/).
 
--   prefer `yarn commit` to `git commit`
--   continuous delivery to NPM
+If the commit messages follow the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0), semantic-release will determine what types of changes are included.
```
Comment (mxdvl) [inline README.md:54]:
```suggestion
If the PR title follows the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0), semantic-release will determine what types of changes are included.
```

**Review by mxdvl — CHANGES_REQUESTED** [review]
I thought that since #53 only the PR title mattered, and PR should only be squashed and merged.

File: README.md (hunk 1 above)
Comment (sndrs) [reply README.md:54]: yeah this is tricky. technically it _is_ the commits themselves – the PR title becomes the commit message when the branch is merged. github is just squashing the other commits for you... will have another look at the wording

Comment (sndrs) [reply README.md]: had another go

**Review by mxdvl — APPROVED** [review]
Very clean now!

**Pull request merged**